app/services/core_data_connector/http/request.rb

```diff
@@ -0,0 +1,39 @@
+require 'typhoeus'
+
+module CoreDataConnector
+  module Http
+    class Request
+
+      DEFAULT_GET_OPTIONS = {
+        followlocation: true
+      }
+
+      def self.delete(url, options = {})
+        parse_response Typhoeus.delete(url, options)
```

Comment:
Nice, I wasn't aware of Typhoeus. Is the advantage here just speed, like it provides lower-level access than something like HTTParty?